### PR TITLE
chore: Release merge 1.1.0

### PR DESCRIPTION
**Pull request by miquelbeltran**
### Description :memo:
- **Purpose**: Merge changes into master for the release
- **Approach**: 

Note to self: remember to merge, not squash!

**Type of change**

- [x] Release process

**Updates**

- See CHANGELOG.md changes

### Test plan :test_tube:

- Run example projects.
- Ensure tests and other CI jobs complete :heavy_check_mark: 

### Author to check :eyeglasses:
- [x] Project and all contained modules builds successfully
- [x] Self-/dev-tested 
- [x] Unit/UI/Automation/Integration tests provided where applicable
- [x] Code is written to standards
- [x] Appropriate documentation written (code comments, internal docs)

### Reviewer to check :heavy_check_mark:
- [ ] Project and all contained modules builds successfully
- [ ] Change has been dev-/reviewer-tested, where possible
- [ ] Unit/UI/Automation/Integration tests provided where applicable
- [ ] Code is written to standards
- [ ] Appropriate documentation written (code comments, internal docs)

### CODE REVIEW

**Review by TheRealAgentK — APPROVED** [review]
Looks good to me, left a comment re a possible rephrasing, but that's not a blocker.

File: examples/express-sample/views/index.ejs
Comment (sumitramanga) [reply examples/express-sample/views/index.ejs:15]: @miquelbeltran Agreed!

File: package-lock.json
Comment (sumitramanga) [inline package-lock.json:5348]: Do we need to action anything here

File: test/raygun_unhandled_rejection_test.js
Comment (sumitramanga) [inline test/raygun_unhandled_rejection_test.js:13]: A code comment here to clarify would be good to add

**Review by sumitramanga — COMMENTED** [review]
Overall, it looks great! Some minor things to consider :)

File: package-lock.json
Comment (miquelbeltran) [reply package-lock.json:5348]: We don't use https://www.npmjs.com/package/read-package-json directly, so I guess this is pulled by a 3rd party package. I don't think this is a blocker.

File: test/raygun_unhandled_rejection_test.js
Comment (miquelbeltran) [reply test/raygun_unhandled_rejection_test.js:13]: I will add it.
FYI this was copied from `test/raygun_uncaught_exception_test.js` which had no comment here either.

**Comment by miquelbeltran** [comment]
Suggestions submitted in this PR: https://github.com/MindscapeHQ/raygun4node/pull/253 Please take a look!

**Review by sumitramanga — APPROVED** [review]
LGTM! Please wait on @PanosNB before merging 💯

**Review by PanosNB — COMMENTED** [review]
Just a small question on my end, all looks good and I can approve after that

**Review by PanosNB — APPROVED** [review]
After discussing with Sumitra, we clarified that users specify a keyword and if it is found, the whole things is skipped. Ok, no concerns then. So not connected to a specific fixed word

**Comment by sumitramanga** [comment]
Tests look to be passing as well locally !

File: examples/express-sample/app.js
Comment (miquelbeltran) [reply examples/express-sample/app.js:60]: Note that what you are looking at is the example project, not the actual package implementation. This is just an example of how to implement a skip, not actual shipped code.

This example code only checks the contents of the error message, and doesn't look at the stacktrace nor other properties of the error payload.

**Comment by miquelbeltran** [comment]
> After discussing with Sumitra, we clarified that users specify a keyword and if it is found, the whole things is skipped. Ok, no concerns then. So not connected to a specific fixed word

Sorry, this is not correct. I have added clarification in a reply to the comment.

Nevertheless, you are looking at an example code, not the package implementation, so this is not a concern for end users.

**Comment by miquelbeltran** [comment]
For further clarification: What users need to do to skip an error is to implement their own `onBeforeSend`, returning `null` when they way to skip an error report.

What the example project contains is just an example on how to implement that, there is no logic shipped with the package where the user "specifies a keyword", users can implement `onBeforeSend` to fit their needs.

**Pull request merged**
